src/ProjectTemplates/Web.ProjectTemplates/Worker-CSharp.csproj.in

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">True</NoDefaultLaunchSettingsFile>
+    <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.Application1</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--#if (WebJobs) -->
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.2" /> <!--TODO Version-->
```

Comment:
Should this be managed via `eng/Versions.props`?

Reply:
I expect so. I wanted to make sure it worked first.